package.json

```diff
@@ -391,6 +391,11 @@
 					"type": "boolean",
 					"default": false,
 					"description": "Whether to automatically send a 'hot reload' request during a Flutter debug session when saving files."
+				},
+				"dart.analyzeAngularTemplates": {
```

Comment:
You're right; the last line is bogus. We include options here that just generally require a reload (because of behaviour at activation time). I've improved the comment in 2a8527608e39d401f56608476a4ccb241ecacf7c.